src/FormatNumber.elm

```diff
@@ -44,7 +44,7 @@ import String
 
     >>> import FormatNumber.Locales exposing (Locale)
     >>> format (Locale 3 "." ",") -7654.3210
-    "−7.654,321"
+    "-7.654,321"
```

Comment:
I believe this was a mixture of an accidental edit and a tiny tiny details: you replaced a `−` by a `-`.

Now I swear I can listen your _WTF_… but worry not. The thing is that [the hyphen (the one you typed) is not the same thing as the minus sign](http://www.punctuationmatters.com/the-difference-between-a-dash-and-a-minus-sign/) (the one the package should use for the output).

Can you use the minus sign instead?